tests/dotnet/MySingleView/AppDelegate.cs

```diff
@@ -7,9 +7,21 @@ namespace MySingleView
 {
 	public partial class AppDelegate : UIApplicationDelegate
 	{
-		public override UIWindow Window {
-			get;
-			set;
+		UIWindow window;
```

Comment:
not a big deal but it would be better to keep the property (for anyone looking for an example)

Reply:
I was planning on reverting to the more complex sample again when I get storyboards working.